# HG changeset patch
# User Pantxo Diribarne <pantxo.diribarne@gmail.com>
# Date 1521038540 -3600
#      Wed Mar 14 15:42:20 2018 +0100
# Node ID a41e264609c0476b00db062459255a3fc1d0c83d
# Parent  7c7b60dd4d4cde2e318905baaa54f6e10ec0af29
Fix misconfigured OCTAVE_DOCDIR (bug #53006)

diff --git a/build-aux/subst-default-vals.in.sh b/build-aux/subst-default-vals.in.sh
--- a/build-aux/subst-default-vals.in.sh
+++ b/build-aux/subst-default-vals.in.sh
@@ -56,31 +56,32 @@ infofile=`echo "@infofile@" | sed "s|^${
 localapifcnfiledir=`echo "@localapifcnfiledir@" | sed "s|^${prefix}/||"`
 localfcnfiledir=`echo "@localfcnfiledir@" | sed "s|^${prefix}/||"`
 localstartupfiledir=`echo "@localstartupfiledir@" | sed "s|^${prefix}/||"`
 localapiarchlibdir=`echo "@localapiarchlibdir@" | sed "s|^${prefix}/||"`
 localverfcnfiledir=`echo "@localverfcnfiledir@" | sed "s|^${prefix}/||"`
 man1dir=`echo "@man1dir@" | sed "s|^${prefix}/||"`
 mandir=`echo "@mandir@" | sed "s|^${prefix}/||"`
 octdatadir=`echo "@octdatadir@" | sed "s|^${prefix}/||"`
-octdatadir=`echo "@octdocdir@" | sed "s|^${prefix}/||"`
+octdocdir=`echo "@octdocdir@" | sed "s|^${prefix}/||"`
 octetcdir=`echo "@octetcdir@" | sed "s|^${prefix}/||"`
 octincludedir=`echo "@octincludedir@" | sed "s|^${prefix}/||"`
 octlocaledir=`echo "@octlocaledir@" | sed "s|^${prefix}/||"`
 octtestsdir=`echo "@octtestsdir@" | sed "s|^${prefix}/||"`
 startupfiledir=`echo "@startupfiledir@" | sed "s|^${prefix}/||"`
 texi_macros_file=`echo "@texi_macros_file@" | sed "s|^${prefix}/||"`
 
 $SED \
   -e "s|%NO_EDIT_WARNING%|DO NOT EDIT!  Generated automatically by subst-default-vals.|" \
   -e "s|%OCTAVE_ARCHLIBDIR%|\"${archlibdir}\"|" \
   -e "s|%OCTAVE_BINDIR%|\"${bindir}\"|" \
   -e "s|%OCTAVE_CANONICAL_HOST_TYPE%|\"${canonical_host_type}\"|" \
   -e "s|%OCTAVE_DATADIR%|\"${datadir}\"|" \
   -e "s|%OCTAVE_DATAROOTDIR%|\"${datarootdir}\"|" \
+  -e "s|%OCTAVE_DOCDIR%|\"${docdir}\"|" \
   -e "s|%OCTAVE_DEFAULT_PAGER%|\"${DEFAULT_PAGER}\"|" \
   -e "s|%OCTAVE_DOC_CACHE_FILE%|\"${doc_cache_file}\"|" \
   -e "s|%OCTAVE_EXEC_PREFIX%|\"${exec_prefix}\"|" \
   -e "s|%OCTAVE_EXEEXT%|\"${EXEEXT}\"|" \
   -e "s|%OCTAVE_FCNFILEDIR%|\"${fcnfiledir}\"|" \
   -e "s|%OCTAVE_IMAGEDIR%|\"${imagedir}\"|" \
   -e "s|%OCTAVE_INCLUDEDIR%|\"${includedir}\"|" \
   -e "s|%OCTAVE_INFODIR%|\"${infodir}\"|" \
diff --git a/libinterp/corefcn/toplev.cc b/libinterp/corefcn/toplev.cc
--- a/libinterp/corefcn/toplev.cc
+++ b/libinterp/corefcn/toplev.cc
@@ -406,16 +406,17 @@ specified option.
            { "localstartupfiledir", octave::config::local_startupfile_dir () },
            { "localverarchlibdir", octave::config::local_ver_arch_lib_dir () },
            { "localverfcnfiledir", octave::config::local_ver_fcn_file_dir () },
            { "localveroctfiledir", octave::config::local_ver_oct_file_dir () },
            { "man1dir", octave::config::man1_dir () },
            { "man1ext", octave::config::man1_ext () },
            { "mandir", octave::config::man_dir () },
            { "octdatadir", octave::config::oct_data_dir () },
+           { "octdocdir", octave::config::oct_doc_dir () },
            { "octfiledir", octave::config::oct_file_dir () },
            { "octetcdir", octave::config::oct_etc_dir () },
            { "octincludedir", octave::config::oct_include_dir () },
            { "octlibdir", octave::config::oct_lib_dir () },
            { "octtestsdir", octave::config::oct_tests_dir () },
            { "release_date", OCTAVE_RELEASE_DATE },
            { "startupfiledir", octave::config::startupfile_dir () },
            { "version", OCTAVE_VERSION }};
